app/presenters/content_item_presenter.rb

```diff
@@ -49,6 +49,11 @@ def content_id
     content_item["content_id"]
   end
 
+  def web_url
+    # Plek.current.website_root + content_item["base_path"]
+    "https://government-frontend-pr-878.herokuapp.com" + content_item["base_path"]
```

Comment:
just a suggestion: You could try to make Plek.current read the env variable from heroku 

https://stackoverflow.com/questions/12570579/how-to-get-heroku-app-name-url-from-inside-the-app